File: Ohtap E Commerce Domain Models/Development/PHP/Objects/Payments/IPaymentData.php
<?php

namespace Ohtap\ECommerce\DomainModels\Objects\Payments;


use  Ohtap\ECommerce\DomainModels\Core\Object as  CoreObjects ;


interface IPaymentData extends IPaymentAmount,IPaymentCurrency,IPaymentType,CoreObjects\IObjectID, CoreObjects\IObjectCode,
    CoreObjects\IObjectDescription, CoreObjects\IObjectName, CoreObjects\IObjectOrder, CoreObjects\IObjectState, CoreObjects\IObjectTitle
{

}

File: Ohtap E Commerce Domain Models/Development/NetCore/Models Objects/Models Objects/Payments/IPaymentData.cs
using Ohtap.ECommerce.Domain.Models.Objects.Datasets;
using Ohtap.ECommerce.Domain.Models.Objects.Currencies;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using CoreObject = Ohtap.ECommerce.Domain.Models.Core.Object;

namespace Ohtap.ECommerce.Domain.Models.Objects.Payments
{
    public interface IPaymentData :  CoreObject.IObjectID, CoreObject.IObjectCode, CoreObject.IObjectName, CoreObject.IObjectOrder, CoreObject.IObjectDescription, CoreObject.IObjectState, CoreObject.IObjectTitle, IPaymentType, IPaymentAmount
    {
        ICurrencyData Currency { get; set; }
        IDataItemList<IDataItemData> Dataset { get; set; }
    }
}
